### ISSUE TITLE
Suggestion: Set a fixed width / height on signs

### ISSUE THREAD

**Issue by ablohme86** [JKPotato-Computer/SignMaker#12]
That would have been great because sometimes I need perfect sizes for over-head signs in my game, it looks very untidy to have 3 signs next to eachother with different heights and widths.

Also another suggestion: Having more than 1 city in each sign allowing direction sign on left or right on each "line".

Example:

 ---------------------
|                     |
|   <- Houston        |
|   -> Dallas.        | 
|   -> Dream Town     |
|                     |
 ---------------------

**Comment by ajh123** [comment]
I agree with the ability to set the size of signs. When I export them I need to make sure the sign fit in a 128 pixel grid to work well with Minecraft maps. For very big signs I need to span multiple maps, so if I can set the width and height to be a multiple of 128 I will know how many maps I need.
